### DISCUSSION TITLE
version 7.28 to be released ASAP

### DISCUSSION THREAD

**Discussion by ddennedy** [mltframework/mlt#1026] [Announcements]
We are aware of some major regressions introduced in the recently released version 7.26 and will be releasing a new version soon to correct that. In addition to #1022 and #1023 there is also f104d17b, an issue with using `melt` to render XML that uses Movit OpenGL effects.

**Comment by j-b-m** [comment]
Thanks for the quick response to the seeking issues!

**Comment by ddennedy** [comment]
This is done
